bin/common.sh

```diff
@@ -121,6 +121,22 @@ JAVA_OPTS+=" ${ZEPPELIN_JAVA_OPTS} -Dfile.encoding=${ZEPPELIN_ENCODING} ${ZEPPEL
 JAVA_OPTS+=" -Dlog4j.configuration=file://${ZEPPELIN_CONF_DIR}/log4j.properties"
 export JAVA_OPTS
 
+if [[ -n "${ZEPPELIN_JMX_ENABLE}" ]]; then
+  if [[ -n "${ZEPPELIN_JMX_OVERRIDE}" ]]; then
+    JAVA_OPTS="${ZEPPELIN_JMX_OVERRIDE} ${JAVA_OPTS}"
```

Comment:
would `ZEPPELIN_JMX_OVERRIDE` this get abused for random non-JMX related JAVA OPTS?

Reply:
Yes, it could be. I, however, think we need to provide override way at the same time. Do you have any better idea to handle it? Without it, It's better to remove this feature this time and add when it's needed.

Reply:
Removed

Reply:
hmm, not really - unless we have some ways/formats to check against.